src/mono/wasm/runtime/polyfills.ts

```diff
@@ -201,15 +201,19 @@ export async function init_polyfills_async(): Promise<void> {
             globalThis.crypto = <any>{};
         }
         if (!globalThis.crypto.getRandomValues) {
-            const nodeCrypto = INTERNAL.require("node:crypto");
-            if (nodeCrypto.webcrypto) {
-                globalThis.crypto = nodeCrypto.webcrypto;
-            } else if (nodeCrypto.randomBytes) {
-                globalThis.crypto.getRandomValues = (buffer: TypedArray) => {
-                    if (buffer) {
-                        buffer.set(nodeCrypto.randomBytes(buffer.length));
-                    }
-                };
+            try {
+                const nodeCrypto = INTERNAL.require("node:crypto");
+                if (nodeCrypto.webcrypto) {
+                    globalThis.crypto = nodeCrypto.webcrypto;
+                } else if (nodeCrypto.randomBytes) {
+                    globalThis.crypto.getRandomValues = (buffer: TypedArray) => {
+                        if (buffer) {
+                            buffer.set(nodeCrypto.randomBytes(buffer.length));
+                        }
+                    };
+                }
+            } catch (err: any) {
+                // Noop as node can be delivered without "node:crypto", but it's possible that we won't need it.
```

Comment:
should we instead install polyfil which would throw better explanation ? Or at least log it into console.